src/callbacks_stage/entropy_bounded_limiter_1d.jl

```diff
@@ -35,14 +35,8 @@ function limiter_entropy_bounded!(u, u_prev, exp_entropy_decrease_max,
         # Limiting only if entropy DECREASE below a user defined threshold is detected.
         d_exp_s_min < exp_entropy_decrease_max || continue
 
-        # Compute mean value
-        u_mean = zero(get_node_vars(u, equations, dg, 1, element))
-        for i in eachnode(dg)
-            u_node = get_node_vars(u, equations, dg, i, element)
-            u_mean += u_node * weights[i]
-        end
-        # Note that the reference element is [-1,1]^ndims(dg), thus the weights sum to 2
-        u_mean = u_mean / 2^ndims(mesh)
+        u_mean = compute_u_mean(u, mesh, equations, dg, weights,
+                                element)
```

Comment:
Could you please use an ordering of the arguments similar to what we typically use in other places (such as the `flux_differencing_kernel!`), i.e.,

```suggestion
        u_mean = compute_u_mean(u, element, mesh, equations, weights, dg)
```

?